internal/operator/clusterutilities.go

```diff
@@ -166,7 +166,6 @@ type DeploymentTemplateFields struct {
 	ScopeLabel          string
 	//next 2 are for the replica deployment only
 	Replicas                 string
-	PrimaryHost              string
```

Comment:
Is this comment still accurate/necessary?

Reply:
the `Replicas` one is still accurate.